temporalcli/client.go

```diff
@@ -172,6 +173,11 @@ func (c *ClientOptions) dialClient(cctx *CommandContext) (client.Client, error)
 	if err != nil {
 		return nil, fmt.Errorf("failed creating client options: %w", err)
 	}
+
+	if grpcAuthorityExplicitlySet {
```

Comment:
```suggestion
	if c.GrpcAuthority != "" {
```

Don't need the flag-changed stuff above. That's for an advanced client env config stuff.